Krzywicki_reflection_1.qmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "Self-reflection-1-krzywicki"
+format: html
+---
+
+<span style="color: blue;">Questions About you
+
+Please tell me a little bit about your research interests and how you imagine integrating spatial data in your research.
+
+I am studying how global land use change affects local sense of place through the lens of migratory networks. I envision using spatial data to compare and contrast sites along migratory bird networks.
+
+How do you learn best, and what do you feel is your biggest strength as a student?
+
+I am more of a words person than a visual learner, so maps don't always come the easiest to me! I am excited to push myself this semester.
+
+If you want to, you can set a goal for this semester for something you want to work on in the context of this course (outside of the topic matter itself) -- for example, using StackOverflow to get answers to analysis questions or using git to track progress on homeworks. If you choose to set this type of goal, what can you do to help support yourself in working toward that goal? How can I help you to meet this goal?
+
+You can, if you choose, also let me know if there is anything else you have going on this semester that you would like me to know about (e.g., field work, other courses, prepping your qualifying exam, working outside of assistantship).
+
+This Course
+
+Please tell me a little about your prior experience with R ("I use it all the time", "I am still learning", "I've never used it before"). What do you like most about working with R? What are the biggest obstacles for you when working with R?
+
+I used R in Biometry and Eco Stats last year. Those were my first exposures to it. I would describe myself as "familiarish but not confident". My favorite thing about it so far is the powerful and flexible data visualization features. As someone who spent years making graphs in excel for reports, I really appreciate ggplot. 
+
+I think my biggest obstacle is syntax issues and remembering the steps I need to take to get things done. I could be a better annotator.
```

Comment:
Hopefully, the short weekly homeworks will help build some muscle memory around syntax!